python/dgl/graph_index.py

```diff
@@ -915,7 +915,13 @@ class NodeFlowIndex(GraphIndex):
         The offsets of the flows.
     """
     def __init__(self, handle, parent, node_mapping, edge_mapping, layers, flows):
-        super(NodeFlowIndex, self).__init__(handle, parent.is_multigraph(), parent.is_readonly())
+        if parent is None:
+            super(NodeFlowIndex, self).__init__(handle, False, True)
+        else:
+            super(NodeFlowIndex, self).__init__(
+                handle,
+                parent.is_multigraph(),
+                parent.is_readonly())
```

Comment:
@zheng-da does NodeFlow support non-readonly mode right now?

Reply:
NodeFlow is readonly right now.

Reply:
Another question is if the parent is None, what happens when `copy_to/from_parent` is called?

Reply:
Will it be changed in the future?

Reply:
A user should get `NodeFlow` whose `parent` is None. This is a tempoary solution because we don't have a distributed DGLGraph. Once we have it, we can set `parent` correctly.